manifest.json

```diff
@@ -10,7 +10,8 @@
       "repo": "kubewarden/ui",
       "versions": [
         "1.0.0",
-	"1.0.1"
+	      "1.0.1",
```

Comment:
@jordojordo Can you fix the indentation here

Reply:
@nwmac Sorry I'm a little confused, should there be no indentation past the "1.0.0" version?